<launch>

<node name="aitsmc" pkg="usv_ros" type="aitsmc" >
    <param name = "k_u" value = "1.1" />
	<param name = "k_r" value = "1.1" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_r" value = "0.01" />
	<param name = "k2_u" value = "0.001" />
    <param name = "k2_r" value = "0.001" />
	<param name = "mu_u" value = "0.01" />
	<param name = "mu_r" value = "0.01" />
	<param name = "tc_u" value = "2" />
	<param name = "tc_r" value = "2" />
    <param name = "q_u" value = "3" />
	<param name = "q_r" value = "3" />
    <param name = "p_u" value = "5" />
	<param name = "p_r" value = "5" />
</node>

</launch>